xml/System.Net.Http/HttpClient.xml

```diff
@@ -112,37 +112,14 @@ On each platform, <xref:System.Net.Http.HttpClient> tries to use the best availa
 Users can also configure a specific transport for <xref:System.Net.Http.HttpClient> by invoking the <xref:System.Net.Http.HttpClient.%23ctor*> constructor that takes an <xref:System.Net.Http.HttpMessageHandler>.
 
 ### HttpClient and .NET Core
-
 Starting with .NET Core 2.1, the <xref:System.Net.Http.SocketsHttpHandler?displayProperty=nameWithType> class instead of `HttpClientHandler` provides the implementation used by higher-level HTTP networking classes such as `HttpClient`. The use of <xref:System.Net.Http.SocketsHttpHandler> offers a number of advantages:
-
 - A significant performance improvement when compared with the previous implementation.
-
 - The elimination of platform dependencies, which simplifies deployment and servicing. For example, `libcurl` is no longer a dependency on .NET Core for macOS and .NET Core for Linux.
-
 - Consistent behavior across all .NET platforms.
 
-If this change is undesirable, you can configure your application to use the older <xref:System.Net.Http.HttpClientHandler?displayProperty=nameWithType> instead in a number of ways:
-
-- By calling the <xref:System.AppContext.SetSwitch%2A?displayProperty=nameWithType> method as follows:
+### Configure behavior using run-time configuration options
 
-   ```csharp
-   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", false);
-   ```
-
-   ```vb
-   AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", False)
-   ``` 
-- By defining the `System.Net.Http.UseSocketsHttpHandler` switch in the *.netcore.runtimeconfig.json* configuration file:
-
-  ```json
-  "runtimeOptions": {
-    "configProperties": {
-        "System.Net.Http.UseSocketsHttpHandler": false
-    }
-  }
-  ```
-  
-- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
+Certain aspects of <xref:System.Net.Http.HttpClient>'s behavior are customizable through [Run-time configuration options](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/networking). However, the behavior of these switches differs through .NET versions. For example, in .NET Core 2.1 - 3.1, you can configure whether <xref:System.Net.Http.SocketsHttpHandler> is used by default, but that option is no longer available starting in .NET 5.0.
```

Comment:
@gewarren is this the right way to reference conceptual docs?
```
[Run-time configuration options](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/networking)
```

Reply:
No, it should be `[Run-time configuration options](/dotnet/core/run-time-config/networking)`. Thanks for checking.